src/svc.c

```diff
@@ -230,39 +230,13 @@ svc_init(svc_init_params *params)
 	return true;
 }
 
-struct rpc_msg *
-alloc_rpc_msg(void)
-{
-	struct rpc_msg *msg = mem_alloc(sizeof(struct rpc_msg));
-
-	TAILQ_INIT_ENTRY(msg, msg_q);
```

Comment:
It hasn't moved. It is never used anywhere. I'm trying to figure out what you meant, but see nothing in the rpc_call_ctx other than the rb tree.

In the ganesha nfs_core.h, there is a struct glist_head req_q; is that what you mean?